cli/awsbatch/awsbsub.py

```diff
@@ -94,6 +94,13 @@ def _get_parser():
         "if they have not finished. It must be at least 60 seconds",
         type=int,
     )
+    # MNP parameter
+    parser.add_argument(
+        "-n",
+        "--nodes",
+        help="The number of nodes to reserve for the job. " "It enables Multi Node Parallel submission",
```

Comment:
You can remove the internal `" "`

Reply:
Right! The autoformat command is not as smart as I thought :)